bottlecap/src/metrics/dogstatsd.rs

```diff
@@ -0,0 +1,36 @@
+use std::sync::mpsc::Sender;
+
+use crate::events::{self, Event, MetricEvent};
+
+pub struct DogStatsD<'a> {
+    host: &'a str,
+    port: u16,
+}
+
+impl<'a> DogStatsD<'a> {
+    pub fn new(host: &'a str, port: u16) -> Self {
```

Comment:
Similar comment about spawning the EventBus. Don't think this needs to be tied to a struct even. Do we even need this "host" field?